### PR TITLE
Add kleva adapter

### PR DESCRIPTION
**Pull request by graham01u**
##### Twitter Link: 
https://twitter.com/kleva_protocol

##### List of audit links if any: 
https://docs.kleva.io/audit-report/peckshield
https://docs.kleva.io/audit-report/sooho
https://docs.kleva.io/audit-report/scvsoft

##### Website Link: 
https://kleva.io/

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
N/A

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
N/A

##### Short Description (to be shown on DefiLlama):
KLEVA Protocol is the first Leveraged Yield Farming Protocol on KLAYTN, with aim to become the largest Lending Protocol for Leveraged Yield Farmers and Lenders. 

##### Token address and ticker if any:
KLEVA: 0x5fff3a6c16c2208103f318f4713d4d90601a7313

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Yield

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
N/A

##### forkedFrom (Does your project originate from another project):
Alpaca Finance

##### methodology (what is being counted as tvl, how is tvl being calculated):
TVL is counted as the sum of Lending TVL and Farming TVL

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/kleva exports TVL: 
        
 
 
    klaytn                    251.69 M
    
    total                    251.69 M

File: projects/kleva/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const retry = require('async-retry')
+const axios = require("axios");
+const BigNumber = require("bignumber.js");
+const { toUSDTBalances } = require('../helper/balances');
+
+async function fetchLiquidity() {
+  const tvl = await retry(async bail => await axios.get('https://bbs5kn3x0j.execute-api.ap-southeast-1.amazonaws.com/dev/klay/ksInfo/tvl'))
```
Comment (g1nt0ki) [inline projects/kleva/index.js:7]: Hi, can you swap this with calls to klaytn chain for calculating tvl, let me know if you need some help

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

File: projects/kleva/index.js (hunk 1 above)
Comment (graham01u) [reply projects/kleva/index.js:7]: Could you elaborate it in detail? This call heads to AWS Lambda which is connected to Klaytn chain already. What do you mean "swap this with calls to klaytn chain"?

Comment (g1nt0ki) [reply projects/kleva/index.js:7]: @graham01u "AWS Lambda which is connected to Klaytn chain" is this part is opaque and creates dependency on this endpoint working/maintained. But the values are fetched by making the calls against chain rpc from our adapter means we have more control, also would be able to fetch historical data. Let me know if you need some help with setting up the adapter :)

**Comment by g1nt0ki** [comment]
@graham01u did you have a chance to look into this?

**Pull request closed without merging**